cli/src/pcluster/validators/ec2_validators.py

```diff
@@ -422,10 +422,10 @@ def _validate_with_subnets(
 
         if not found_qualified_capacity_reservation:
             self._add_failure(
-                "Queue {queue} uses subnets in these availability zones: {subnet_azs_without_reservations} but the "
-                "Capacity Reservation Resource Group ({cr_group_arn}) has reservations in these "
-                "availability zones: {cr_azs}. You can either add a capacity reservation in the "
-                "availability zones that the subnets are in or remove the Capacity Reservation from the "
+                "Queue {queue} has a subnet configuration mapping to the following availability zones: "
```

Comment:
wrap {queue}, {subnet_azs_without_reservations} and all the other values with single quote, e.g. '{queue}'